lib/ConnectionContextBase.ts

```diff
@@ -141,7 +140,7 @@ export module ConnectionContextBase {
     if (userAgent.length > Constants.maxUserAgentLength) {
       throw new Error(
         `The user-agent string cannot be more than 128 characters in length.` +
-        `The given user-agent string is: ${userAgent} with length: ${userAgent.length}`
+          `The given user-agent string is: ${userAgent} with length: ${userAgent.length}`
```

Comment:
Was this intentional?

Reply:
Its just the prettier formatter extension